gallery/visualtests/pageLayouts.js

```diff
@@ -23,6 +25,11 @@ describe('Page Layout', function() {
       await browser.eyesSnapshot(null);
     });
 
+    it('passes a11y checks with sidebar, system notice, and page scrolling', async function() {
```

Comment:
In these pages I chose to only test the busiest examples: The legacy one with every element and page scrolling, the equivalent legacy one but with section scrolling, and the global sidebar one with every element.